guidesmiths/systemic-aws-s3
# Setting up commitlinter

### What’s the focus of this PR

Add 2 packages (husky and commitlint), to perform best practices validation before performing a commit.

### How to review this PR

* Check file changes
* Install dependencies with `npm install` command
* Create any file and commit without the best practices.

### Related work items
- #17 
![commitlint](https://user-images.githubusercontent.com/10531605/135686710-04525f35-007c-4a76-a2e4-eccea33ca963.gif)


### Before submitting this PR, I made sure:
- [ ] I've added tests
